profile/firebase.json

```diff
@@ -0,0 +1,11 @@
+{
+  "emulators": {
+    "firestore": {
+      "port": 8090
+    },
+    "ui": {
+      "enabled": true,
+      "port": 4000
+    }
+  }
+}
```

Comment:
Do every lambda needs this config file?

Reply:
Yes for testing

Reply:
Can we have it in a single file and use it from there